v3/mqtt_dummy.go

```diff
@@ -38,9 +41,16 @@ var (
 	DefaultUsers    = []string{"good", "1234"}
 	DefaultSubData  []byte
 
+	DefaultItems = []*KeyValue{
+		&KeyValue{Key: "key1", Value: "value1", ModificationTime: time.Now()},
+		&KeyValue{Key: "key2", Value: "value2", ModificationTime: time.Now()},
+	}
+
 	currentDevice = "" // XXX - fix the logic that stores this globally
 
 	globalConf DummyConfig
+	packetID   uint32 = 1
+	kvRevision uint32 = 1
```

Comment:
Ah global variable, So it needs atomic operation.

Reply:
Yes, not so good. This was a simple test server, but it kept getting bigger. Maybe next change, I will refactor it.